oasislmf/utils/defaults.py

```diff
@@ -59,6 +64,47 @@
     if cov_type in ['buildings', 'other', 'contents', 'bi']
 })
 
+SOURCE_FILENAMES = OrderedDict({
```

Comment:
Looking at the `SOURCE_FILENAMES` dict:

    SOURCE_FILENAMES = OrderedDict({
        'loc': 'source_location.csv',
        'acc': 'source_account.csv',
        'info': 'source_info.csv',
        'scope': 'source_scope.csv'
    })

Is there is a reason to prefix the filenames with `source`? It is probably better to use the naming scheme for det. loss workflow

* locations file ->`location.csv`
* accounts file -> `account.csv`
* RI info file -> `reinsinfo.csv`
* RI scope file -> `reinsscope.csv`